src/Orneholm.CognitiveWorkbench.Web/Services/ImageComputerVisionAnalyzer.cs

```diff
@@ -67,6 +70,7 @@ public async Task<ComputerVisionAnalyzeResponse> Analyze(string url, string anal
 
                 AnalysisResult = imageAnalysis.Result,
                 OcrResult = recognizedPrintedText.Result,
+                RecognizeTextOperationResult = recognizedText?.Result,
```

Comment:
No need for the `?` what I can see.

Reply:
You're right, as I changed the output of the call, I will remove it